src/pages/internships/index.js

```diff
@@ -0,0 +1,166 @@
+import * as React from 'react'
+import { graphql } from 'gatsby'
+import Layout from '../../components/Layout/Layout'
+import Diagonal from '../../components/Layout/Diagonal/Diagonal'
+import TeamButton from '../../components/TeamButton/TeamButton'
+import ImageOutline from '../../components/ImageOutline/ImageOutline'
+import { Container, Row, Col } from 'react-bootstrap'
+import { StaticImage } from 'gatsby-plugin-image'
+import Title from '../../components/UI/Title/Title'
+import BrandButton from '../../components/UI/BrandButton/BrandButton'
+import { headerBackgroundAlignRight } from "./internships.module.css";
+import '../../styles/main.scss'
+import internWhy from '/src/images/intern-why.jpg';
+
+const InternshipsPage = ({ data }) => {
+  return (
+    <Layout pageTitle="Internships">
+      
+      {/* Header section */}
+      <Container fluid className={`mobile-hide-bg ${headerBackgroundAlignRight}`}>
+        <Container className='mb-5'>
+          <Row>
+            <Col md={7}>
+              <Title className="text-uppercase">Looking for an Internship?</Title>
```

Comment:
@zmm2025 I think you have learned this since you pushed this, but I changed these `<Title>`'s to have a class of `text-uppercase` instead of capitalizing all of the letters.